Test/CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ function(addtest_flags TEST_NAME FLAGS TEST_SOURCE)
 		LINK_FLAGS ${INCREMENTAL}
 		LINKER_LANGUAGE C
 	)
+	set_property(TEST ${TEST_NAME} PROPERTY SKIP_RETURN_CODE 77)
```

Comment:
What does this do?

Reply:
If the return value of the test is `77`, the test is marked as skipped.
https://www.scivision.dev/cmake-skip-test-code-77/

Reply:
Oh, interesting.  For other tests that don't work in some configurations, we just don't run them and rely on the number of executed tests being smaller than the total set.

Reply:
The thing is that it is easier run a unit test and return `77` based on a condition, then do a test in CMake if `__GNUSTEP_MSGSEND__` is set. We *could* streamline platform detection by moving everything into CMake configuration.

Reply:
> we just don't run them 

Has the disadvantage of not seeing if a test was skipped.

Reply:
Yup, I agree that this is better, just wondering if we should do the same thing with other tests...